### DISCUSSION TITLE
Step 4 Preview Data

### DISCUSSION THREAD

**Discussion by edchanonline** [earthlab-education/Earth-Analytics-2023-01-Intro#293] [Q&A03 Python]
Just for awareness - 

```
# Take a look at the data. What got downloaded?
for i, line in enumerate(water_data.content.splitlines()[:35]):
    # Skip commented lines
    if not line.startswith(b'#'):
        print(i, line)
water_data.content
```

I would update `35` to `50` to preview the data. There are 36 comment lines in my data at https://waterdata.usgs.gov/nwis/dv?cb_00010=on&cb_00010=on&cb_00060=on&cb_00095=on&cb_80154=on&cb_80155=on&cb_90095=on&format=rdb&site_no=06610000&legacy=&referred_module=sw&period=&begin_date=1928-10-01&end_date=2022-09-30

**Comment by eculler** [comment]
Nice catch, @eggvoice
